spec/controllers/users_controller_spec.rb

```diff
@@ -459,4 +456,39 @@
       end
     end
   end
+
+  context 'GDS User CSV download' do
+    render_views
+    before(:each) do
+      gdsuser_stub_auth
+    end
+
+    let(:rp_team) { create(:team, team_type: 'rp') }
+
+    describe '#index' do
+      it 'returns success if request format is CSV' do
+        get :index, format: :csv
+        expect(response).to have_http_status(:success)
+        expect(response.header['Content-Type']).to include 'text/csv'
+      end
+
+      it 'get expected headers in CSV' do 
```

Comment:
There are spaces at the end of the line in some of these `get expected headers in CSV`